ceph_cfg/ops_cluster.py

```diff
@@ -58,3 +58,73 @@ def status_refresh(self):
                         output["stderr"])
                         )
         self.model.cluster_status = json.loads(output["stdout"].strip())
+
+
+    def _df_node_to_osd_dict(self, node):
+        node_type = node.get("type")
+        if node_type != "osd":
+            return None
+        osd_id = node.get("id")
+        osd_name = node.get("name")
+        osd_weight_crush = node.get("crush_weight")
+        osd_weight = node.get("reweight")
+        osd_kb_total = node.get("kb")
+        osd_kb_used = node.get("kb_used")
+        osd_kb_avail = node.get("kb_avail")
+        osd_pgs = node.get("pgs")
+        osd_utilization = node.get("utilization")
+        return {
+            "id" : osd_id,
```

Comment:
Temporary variables provide little advantage without error checking for None. The only real advantage is code clarity.

As for list comprehension, This would make sense if the input and output fields where identical.

I felt that particularly "crush_weight" was more ceph implementation specific, and "weight_crush" being presented to the user (and stored in the model) was easier to understand since it would (in default representation of json by python it is sorted alphabetically) be next to "crush_weight".

Which would you prefer?

I add a comment explaining the rational, or move to list comprehension?